src/index.html

```diff
@@ -5,9 +5,43 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Handjet:wght@200;400;700&family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img class="card__img"
+        src="images/imac.jpeg"
+        alt="card-image"
+      >
```

Comment:
```suggestion
      <img 
        class="card__img"
        src="images/imac.jpeg"
        alt="card-image"
      >
```
Fix formatting. Also, alt attribute should be more descriptive
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/3be97a99-16bb-49e8-a0e5-3fc228b2c2ab)